### ISSUE TITLE
[Feature Request] Re-ordering of the viewer list of the dashboard 

### ISSUE THREAD

**Issue by CKY-** [crowbartools/Firebot#1768]
**Describe the solution you'd like**
Please make it possible for the end user to set the order of the viewer list by category.
this would make it possible to place mods and streamer blow the viewer list and vip above, 
or mods vip and viewers ,
or any combination thereof  

**Additional context**
CC:Discord: Nezquickz
Is there a way to change the order of the Broadcaster,Viewers,Mods so its Broadcaster first Mods then viewers?

**Comment by Nezquickz** [comment]
Yeah so basically just being able to rearrange the list.. Or have it like Broadcaster,Mods,Bots,Viewers.

**Comment by CKY-** [comment]
Bots Is not a category just so you know, the only categories are Broadcaster, Moderator, VIP, Artist* and viewer 


* this is new and not in the bot yet, there will be a FR for it.

**Comment by Nezquickz** [comment]
Ah ok i get you... Thanks again...
